input/templates/pop.hbs

```diff
@@ -58,6 +58,9 @@
               <div class="jumbotron-row">
                 {{> catalog/jumbotron jumbotron=content.jumbotron}}
               </div>
+              {{!-- <div class="search-results-row">
+                {{> catalog/search-result search-result=content.searchResult}}
+              </div> --}}
               <div class="col-xs-4 hidden-xs text-left">
```

Comment:
Okay but then can we enable it back? Otherwise it won't work in the implementation side.